RacingAidWpf/ViewModel/MainWindowViewModel.cs

```diff
@@ -14,9 +14,10 @@ public sealed class MainWindowViewModel : NotifyPropertyChanged
     private readonly TimesheetConfigSection timesheetConfigSection = ConfigSectionSingleton.TimesheetSection;
     private readonly RelativeConfigSection relativeConfigSection = ConfigSectionSingleton.RelativeSection;
     private readonly TelemetryConfigSection telemetryConfigSection = ConfigSectionSingleton.TelemetrySection;
-    
-    private readonly OverlayController overlayController;
-    
+
+    // Exposing this as public for the Tests
+    public readonly OverlayController overlayController;
+
```

Comment:
I'd prefer we had a 2nd ctor for testing rather than exposing fields that don't need to be exposed